core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator_test.go

```diff
@@ -588,6 +589,74 @@ func TestCoordinator_ReportIsOnchain(t *testing.T) {
 
 }
 
+func TestCoordinator_ConfirmationDelays(t *testing.T) {
+	t.Run("valid output", func(t *testing.T) {
+		expected := [8]uint32{1, 2, 3, 4, 5, 6, 7, 8}
+		ret := [8]*big.Int{}
+		for i, delay := range expected {
+			// assuming little endian
```

Comment:
Not sure what you mean by this comment - `big.Int` is big-endian (and so is the EVM) but I don't think it's relevant to this test right?

Reply:
Ah this was a copy and paste error :p will remove.